COLLABORATOR_GUIDE.md

```diff
@@ -227,3 +228,35 @@ messages. However, you are only allowed to force push to any Node.js
 branch within 10 minutes from your original push. If someone else
 pushes to the branch or the 10 minute period passes, consider the
 commit final.
+
+### Long Term Support
+
+#### What is LTS?
+
+Long Term Support (often referred to as LTS) is the process of 
+backporting non breaking changes to older versions of Node.js. LTS
+releases guarantee application developers a 30 month support cycle for 
+their products (18 months in active and 12 months maintenance only).
+
+You can find the full LTS plan [online](https://github.com/nodejs/lts#lts-plan).
+
+#### What will be backported?
+
+TLDR; Anything that is non breaking (other than severe security vulnerabilities)
+
+More specifically:
+
+* Documentation
+* Changes to Tests
```

Comment:
Nit: On this and subsequent lines, to not capitalize words after the first. (*Changes to tests* instead of *Changes to Tests*, for example.)